Dockerfile

```diff
@@ -1,33 +1,58 @@
 FROM python:3.6.5-jessie
 
 env DEBIAN_FRONTEND noninteractive
-RUN apt-get update && \
-    apt-get -y install \
+RUN echo 'deb http://deb.debian.org/debian jessie-backports contrib non-free main' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends \
+            build-essential \
+            chromedriver \
+            chrpath \
             dbus-x11 \
+            ffmpeg \
             libdbus-1-dev \
+            libdbus-glib-1-dev \
             libespeak1 \
+            libffi-dev \
+            libfontconfig1 \
+            libfontconfig1-dev \
+            libfreetype6 \
+            libfreetype6-dev \
             libnotify-cil-dev \
+            libssl-dev \
+            libxft-dev \
             notification-daemon \
+            python3-dbus \
```

Comment:
Like before, either build libraries which need python from source or use debian base image instead of python. This will install python3 as debian still uses python3.5 instead of python3.6 by default

Reply:
Oh. I did not realize this. Thanks. This should decrease the image size.